webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+/* eslint-disable */
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+/* eslint-enable */
+module.exports = {
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/i,
+        type: 'asset/resource',
+      },
+    ],
+  },
+
+  //   optimization: {
+  //     runtimeChunk: "single",
+  //   },
```

Comment:
- [ ] @JakariaJishan it's not good practice to have commented code in your codebase kindly remove the commented code, also try and resolve the linter error in line 3 it's not recommended to disable it unless its the last option.

Reply:
I tried but could not resolve the linter error in line 3. so I am with the last option.